.github/workflows/tests.yml

```
@@ -0,0 +1,78 @@
+              - 'examples/**'
+    outputs:
+      website: ${{ steps.filter.outputs.website }}
+      typegate: ${{ steps.filter.outputs.typegate }}
+      full: ${{ steps.filter.outputs.full }}
+
+  pre-commit:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: metatypedev/setup-ghjk@318209a9d215f70716a4ac89dbeb9653a2deb8bc
+      - uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)